src/ngx_http_v2_push_module.c

```diff
@@ -731,6 +746,10 @@ ngx_http_v2_push(ngx_http_request_t *r, u_char *u_str, size_t u_len)
         return NGX_OK;
     }
 
+    if (ngx_http_v2_push_above_max_pushes(h2pscf, h2c)) {
```

Comment:
Probably best to put this before the other check since it's less expensive. Not too important though.